apps/frontend-pwa/package.json

```diff
@@ -17,18 +17,31 @@
     "@fortawesome/free-solid-svg-icons": "6.1.2",
     "@fortawesome/react-fontawesome": "0.2.0",
     "@klicker-uzh/graphql": "^2.0.0-alpha.8",
-    "@uzh-bf/design-system": "0.0.26",
+    "@radix-ui/react-progress": "^1.0.0",
```

Comment:
remove the ^ in deps

Reply:
also, progress should be part of the design system already?

Reply:
removed unused dependency